tests/tracking/test_fluent.py

```diff
@@ -387,12 +387,13 @@ def test_start_run_existing_run(empty_active_run_stack):
     mock_run.info.lifecycle_stage = LifecycleStage.ACTIVE
 
     run_id = uuid.uuid4().hex
+    mock_get_store = mock.patch("mlflow.tracking.fluent._get_store")
```

Comment:
Why do we need to mock this function across the test cases?

Reply:
Because these tests also mock the run object and the `get_run` function, the call to `_get_store().update_run_info` added in this PR would error out because the run doesn't actually exist (in the store).

Reply:
Got it!